inc/ui/class-limits-element.php

```
@@ -236,14 +234,14 @@ public function setup_preview(): void {
 	 *
 	 * @param array       $atts Parameters of the block/shortcode.
 	 * @param string|null $content The content inside the shortcode.
-	 * @return string
+	 * @return void
 	 */
-	public function output($atts, $content = null) {
+	public function output($atts, $content = null): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$content'. (Unused Code Rules)

(UnusedFormalParameter)